src/UBAFeeCalculator/UBAFeeSpokeCalculatorAnalog.test.ts

```diff
@@ -0,0 +1,182 @@
+import { BigNumber } from "ethers";
+import UBAConfig from "./UBAFeeConfig";
+import { getEventFee, getDepositFee, getRefundFee } from "./UBAFeeSpokeCalculatorAnalog";
+import { toBN } from "../utils";
+import { parseEther } from "ethers/lib/utils";
+
+describe("UBAFeeSpokeCalculatorAnalog", () => {
+  const defaultConfig = new UBAConfig(
+    {
+      default: toBN(0),
+    },
+    {
+      default: [
+        [toBN(0), toBN(0)],
+        [toBN(0), parseEther("2")],
```

Comment:
Can you add a comment about this curve? It looks like you want to hardcode 200% as the fee for every `amount`?

Reply:
Also, isn't this curve illegal because there are duplicates `x` points with `x=0`?

Reply:
Resolved these two: https://github.com/across-protocol/sdk-v2/pull/342/commits/2d38b389a3cd4462cdf176a628355226be7b3876